dist/mapdc.css

```diff
@@ -867,7 +867,7 @@ body {
     box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.4);
     pointer-events: none; }
   .dc-chart .map-polygon-shape {
-    pointer-events: auto; }
+    pointer-events: none; }
```

Comment:
Can you just document what this solves?

Reply:
Noting as a comment here that it disables map rotation by preventing the pointer-events for the object with this class. When map rotation is not disabled, right-click dragging is the relevant event source on desktop and pinch on touch-enabled devices.